utils/fs/fs_test.go

```diff
@@ -116,6 +116,68 @@ func TestMountAlreadyMounted(t *testing.T) {
 	}
 }
 
+func TestMountPoint(t *testing.T) {
+	if len(disk.Partitions) < 1 {
+		t.Fatal("Count partitions must be at least 1")
+	}
+
+	type testData struct {
+		mountPointFirst  string
+		mountPointSecond string
+		compare          func(string, string) bool
+		skipUnmountError bool
```

Comment:
In test, compare first mount point and second mount point. If they are the same, umount only once.

Reply:
Done